src/backend/access/appendonly/aosegfiles.c

```diff
@@ -49,6 +49,13 @@
 #include "utils/numeric.h"
 #include "utils/visibility_summary.h"
 
+#define VALIDATE_GP_ROLE() \
+	if (Gp_role != GP_ROLE_DISPATCH) \
+		ereport(ERROR,\
+				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),\
+				 errmsg("Function '%s' can run only on dispatcher.",\
+						PG_FUNCNAME_MACRO)));
```

Comment:
Error messages usually start with a lowercase letter and don't have a period at the end. Maybe reword this message in a better way? For example:
```suggestion
#define VALIDATE_GP_ROLE() \
	if (Gp_role != GP_ROLE_DISPATCH) \
		ereport(ERROR, \
				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED), \
				 errmsg("function '%s' must be executed on the dispatcher", \
						PG_FUNCNAME_MACRO)));
```

Reply:
Ok, fixed